.test-infra/jenkins/job_beam_PerformanceTests_FileBasedIO_IT.groovy

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+
+// This job runs the file-based IOs performance tests on PerfKit Benchmarker.
+job('beam_PerformanceTests_FileBasedIO_IT') {
+    description('Runs PerfKit tests for file-based IOs.')
+
+    // Set default Beam job properties.
+    common_job_properties.setTopLevelMainJobProperties(delegate)
+
+    // Run job in postcommit every 6 hours, don't trigger every push, and
+    // don't email individual committers.
+    common_job_properties.setPostCommit(
+            delegate,
+            '0 */6 * * *',
+            false,
+            'commits@beam.apache.org',
+            false)
+
+    def pipelineArgs = [
+            tempRoot: 'gs://temp-storage-for-perf-tests',
+            project: 'apache-beam-io-testing',
+            numberOfRecords: '1000000',
+            filenamePrefix: 'gs://temp-storage-for-perf-tests/filebased/${BUILD_ID}/TESTIOIT',
+    ]
+    def pipelineArgList = []
+    pipelineArgs.each({
+        key, value -> pipelineArgList.add("\"--$key=$value\"")
+    })
+    def pipelineArgsJoined = "[" + pipelineArgList.join(',') + "]"
+
```

Comment:
Can you add a manual trigger as well ? (for example, see https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_beam_PostCommit_Java_ValidatesRunner_Flink.groovy#L36)

Reply:
sure, added